src/components/PriceLine.tsx

```diff
@@ -73,7 +76,7 @@ const PriceLine = ({
                     <div className='drag-line' style={{ borderTop: `${lineStyle} ${color} 1px` }} />
                 )}
                 <div className='draggable-area' />
-                <div className='drag-price' style={{ backgroundColor: color, width, opacity }}>
+                <div className='drag-price' style={{ backgroundColor: color, width, opacity, display: isBarrierZero ? 'none' : 'block' }}>
```

Comment:
Can we prevent rendering this element instead of using CSS to hide it?
```
{ !isBarrierZero && <div className=....
```

Reply:
Let me try that